### ISSUE TITLE
Investigate adding BAT support

### ISSUE THREAD

**Issue by octalmage** [octalmage/minum#21]
Currently the [Brave](https://brave.com/) browser is using https://uphold.com as their integrated wallet, which doesn't give users ownership of their tokens. Not giving users ownership of their private key goes against the whole point of cryptocurrencies and I won't seriously use Brave until they fix this.

At the end of the day I would love to be able to support BAT enabled websites today. So lets see how much work it would be to integrate a BAT wallet. 

More info can be found here: 

https://basicattentiontoken.org

**Comment by mjchamplin** [comment]
Crypto? 🤔

**Comment by octalmage** [comment]
Yeah the idea is pretty simple. You deposit BAT tokens into the browser, and the browser pays attention to the websites you visit. At the end of the month it splits your tokens up between those sites. I think it's the most innovative alternative to ads I've seen, and it seems like a more sustainable way to support content creators directly. You should sign up and let me know what you think! 

https://publishers.basicattentiontoken.org

I currently distribute 20 BAT a month, my breakdown is this right now: 

![Screen Shot 2019-04-22 at 11 48 55 AM](https://user-images.githubusercontent.com/142006/56512337-b13eee00-64f4-11e9-9921-d1b873e7606a.png)

I'm enjoying this so far, but I want something other than https://brave.com. 

Anyway, I'd love to hear what you think! I think this is a great opportunity to do something with this project!

**Comment by octalmage** [comment]
and of course we can keep the BAT integration out of the way. I've been using Chrome for work, and Brave for personal stuff. I think it would be interesting if I could use minium for personal browsing. If it has BAT integration I think I'd be happy with the trade off because using minium would be supporting content creators.

**Comment by octalmage** [comment]
Reference: 
* https://www.npmjs.com/package/web3-react
* https://github.com/SwapyNetwork/electron-metamask-boilerplate
* https://blog.aragon.org/electron-metamask-secure-easy-to-use-dapps-5a9987d21034/
